src/main/java/com/uid2/admin/util/KeysetUtil.java

```diff
@@ -20,7 +23,8 @@ public static AdminKeyset lookUpKeyset(int siteId, Map<Integer, AdminKeyset> key
     }
 
     public static Integer getMaxKeyset(Map<Integer, AdminKeyset> keysets) {
-        return keysets.keySet().size();
+        if(keysets.isEmpty()) return 3;
```

Comment:
Can we avoid calling getMaxKeyset() until after we've added the initial 3 keysets?

Reply:
Worst case of having non incrementing keyset ids should not cause any problems.... probably good idea to verify and log error if this is so. but in integ/prod there must be master/refresh/pub key ids so shouldn't be a problem

Reply:
Should we have test coverage? Ideally I'd like to see a test that fails if we returned 2 from here instead of 3.